lib/macho/headers.rb

```diff
@@ -33,7 +33,7 @@ module MachO
     MH_CIGAM => "MH_CIGAM",
     MH_MAGIC_64 => "MH_MAGIC_64",
     MH_CIGAM_64 => "MH_CIGAM_64"
```

Comment:
If this missing trailing comma wasn't pointed out, then I guess we should add the following:

```yaml
Style/TrailingCommaInLiteral:
  EnforcedStyleForMultiline: comma
```